.github/workflows/build_bazel.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 The StableHLO Authors.
+
+name: Bazel Build
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths: ['**.txt', 'WORKSPACE']
+  push:
+    branches: [ main ]
+    paths: ['**.txt', 'WORKSPACE']
+  schedule:
+    # Run buildAndTest workflow once a day
+    - cron:  '0 12 * * *'
+  workflow_dispatch:
+
+# Ensure that only a single job or workflow using the same
+# concurrency group will run at a time. This would cancel
+# any in-progress jobs in the same github workflow and github
+# ref (e.g. refs/heads/main or refs/pull/<pr_number>/merge).
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Build using Bazel build.
+# Use Cached build to improve build times.
+jobs:
+  bazel-build:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: Checkout StableHLO
+      uses: actions/checkout@v2
+
+    # Read LLVM version from `build_tools/llvm_version.txt`
```

Comment:
I think the comments in this file are pretty self-evident, so how about we delete them?